news/issue_772.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Checkpoint interval default value has been set to 250 ps instead of 1 ps.
```

Comment:
I can fix this in the release process but we don't need changelog until we've released 1.0 proper